openelevationservice/server/__init__.py

```diff
@@ -94,4 +97,15 @@ def handle_invalid_usage(error):
         'db': db}
     )
 
+    # TODO: Allow gRPC server to run in parallel if needed
+    # grpc_url = '127.0.0.1:5005'
+    # with app.app_context():
+    #     oes_grpc_server.serve(grpc_url)
+    # log.info("Started gRPC server on {}".format(grpc_url))
+    # grpc_process = Process(
+    #     target=oes_grpc_server.serve,
+    #     args=(grpc_url,)
+    # )
+    # grpc_process.start()
+
```

Comment:
Delete comments if they are not necessary

Reply:
DONE